game-core/src/main/java/games/strategy/triplea/delegate/AbstractEndTurnDelegate.java

```diff
@@ -34,6 +34,7 @@
 import games.strategy.triplea.delegate.remote.IAbstractForumPosterDelegate;
 import games.strategy.triplea.formatter.MyFormatter;
 import games.strategy.triplea.player.ITripleAPlayer;
+import games.strategy.triplea.ui.ObjectiveDummyDelegateBridge;
```

Comment:
Is it normal for delegates to have a dependency on a UI package?  I never really thought about that before; it just seems strange to see a dependency in that direction.  I was going to suggest hosting `ObjectiveDummyDelegateBridge` in the `g.s.triplea.delegate` package, but, looking at the implementation of `ObjectiveDummyDelegateBridge`, I'm not sure that's any better.

Reply:
Same thought went through my mind. I left it in the UI package for now since its probably equally bad. Probably need to eventually move it to some kind of common package that both delegates and ui could use.